esmvaltool/recipes/recipe_hyint.yml

```diff
@@ -0,0 +1,85 @@
+
+  references:
+    - giorgi11jc
+    - giorgi14jgr
+
+  projects:
+    - c3s-magic
+
+datasets:
+  - {dataset: ACCESS1-0,      project: CMIP5, exp: [historical,rcp85], ensemble: r1i1p1, start_year: 2004, end_year: 2006}
+#  - {dataset: ACCESS1-3,      project: CMIP5, exp: rcp85, ensemble: r1i1p1, start_year: 2006, end_year: 2016}
+#  - {dataset: MPI-ESM-MR,      project: CMIP5, exp: rcp85, ensemble: r1i1p1, start_year: 2006, end_year: 2016}
+
+preprocessors:
+  preproc:
+    extract_levels: false
+
+diagnostics:
+  hyint:
+    description: HyInt hydroclimatic indices calculation and plotting
+    variables:
+       pr:
+        preprocessor: preproc
+        reference_dataset: "ACCESS1-0"
+        mip: day
+        field: T2Ds
+    scripts:
+      main:
+        script: hyint/hyint.R
+        # Define whether model data should be regridded 
+        # a) false to keep original resolution
+        # b) set desired regridding resolution in cdo format e.g., "r320x160"
+        # c) "REF" to use resolution of reference model
+        rgrid: false   
+        # Reference normalization period to be used for normalized indices 
+        norm_years: [2004,2006]
+
+        # Select type of plot:
+        # 1) lon/lat maps per individual field/exp/multi-year mean
+        # 2) lon/lat maps per individual field/exp-ref-diff/multi-year mean
+        # 3) lon/lat maps multi-field/exp-ref-diff/multi-year mean
+        # 4) lon/lat maps multifield/exp/multiyear - skipped if using multi-year mean
+        # 11) timeseries over required individual region/exp
+        # 12) timeseries over multiple regions/exp
+        # 13) timeseries with multiple models
+        # 14) summary trend coefficients multiple regions
+        # 15) summary trend coefficients multiple models
+        # plot_type: [2, 3, 12, 13, 14, 15]
+        plot_type: [14]
+        # Select single plot or multi-panel figure
+        multipanel: false
+        # Select number of columns and rows in multi-panel figure
+        npancol: 2 # number of columns for trend/tseries multi-panel figures
+        npanrow: 3 # number of rows for trend/tseries multi-panel figures  
+        # User defined label to be added to trends and figures filename
+        label: "test"
+
+        # Select regions for timeseries and maps from the following list
+        # 1=World, 2=World60 (60S/60N),3=Tropics (30S/30N), 4=South America, 
+        # 5=Africa,6=Northi,7=America,8=India,9=Europe,10=East-Asia,
+        # 11=Australia
+        selregions: [2,4,5,6,7]
+        # Select one or more fields for timeseries and maps from the following 
+        # list (order-sensitive): 
+        # SDII, DSL, WSL, HY-INT, ABS_INT, ABS_DSL, ABS_WSL, PA, R95 
+        selfields: [8, 4, 1, 9, 3, 2]
+       # Select automated or pre-set range of values in plots
+        autolevels: true       
+
+        ## ------- Advanced settings ----------
+        
+        topography_highres: "~/work/data/Elevation/GMTED2010_15n030_0125deg.nc"
```

Comment:
Is this file needed? If so, how do I obtain it?

Reply:
This option was removed for standard users. We may later on include this option by passing a high resolution topography through the preproc